lightly/data/_helpers.py

```diff
@@ -134,10 +60,21 @@ def _load_dataset_from_folder(root, transform):
         Dataset consisting of images in the root directory
     """
 
-    if _contains_subdirs(root):
-        dataset = datasets.ImageFolder(root, transform=transform)
+    contains_videos = _contains_videos(root, VIDEO_EXTENSIONS)
+    if contains_videos and not VIDEO_DATASET_AVAILABLE:
+        raise ValueError(f'The input directory {root} contains videos '
```

Comment:
the error message could be more detailed (it can say it failed because torch version is old or some other package is not installed)